src/vm_interrupts.rs

```diff
@@ -25,6 +25,7 @@ pub type Result<T> = core::result::Result<T, Error>;
 enum BindStatus {
     Binding(CpuId, ImsicFileId),
     Bound(CpuId, ImsicFileId),
+    // Rebinding has old CpuId and old ImsicFileId first and new ones later.
     Rebinding(CpuId, ImsicFileId, CpuId, ImsicFileId),
```

Comment:
optional: you could name the fields to make this more obvious, e.g.:

```
Rebinding { old_cpu: CpuId, old_file: ImsicFileId, new_cpu: CpuId, new_file: ImsicFileId }
```

Reply:
In my opinion I find it better as a tuple. With Rebinding being struct it will create mismatch between enum fields and the way it's accessed through out the code.